components/servo/examples/winit_minimal.rs

```diff
@@ -49,24 +49,15 @@ struct AppState {
 }
 
 impl ::servo::WebViewDelegate for AppState {
-    fn notify_ready_to_show(&self, webview: WebView) {
-        let rect = self
-            .window_delegate
-            .get_coordinates()
-            .get_viewport()
-            .to_f32();
-        webview.focus();
-        webview.move_resize(rect);
-        webview.raise_to_top(true);
-    }
-
     fn notify_new_frame_ready(&self, _: WebView) {
         self.window_delegate.window.request_redraw();
     }
 
     fn request_open_auxiliary_webview(&self, parent_webview: WebView) -> Option<WebView> {
         let webview = self.servo.new_auxiliary_webview();
         webview.set_delegate(parent_webview.delegate());
+        webview.focus();
```

Comment:
in this auxiliary webview delegate, does the webview exist in the constellation at this point? i think it won’t until we return the WebView, the EmbedderMsg::AllowOpeningWebView handler sends the WebViewId back to script, script sends ScriptMsg::ScriptNewAuxiliary, then the constellation receives it?

Reply:
we’ve opened #35672 to address this

Reply:
#35672 has landed, so this should no longer be a problem :)